sources/web/datalab/static/extensions/charting.js

```diff
@@ -215,11 +268,18 @@ define("extensions/charting", function () {
       data = convertListToDataTable(data);
     }
     convertDates(data);
+    model.dom = dom;
 
+    removeStaticChart(model);  // Remove any existing chart PNG.
     require(['visualization!' + chartScript], function (visualization) {
       var chartType = visualization[chartInfo.name];
       var chart = new chartType(dom);
 
+      // Generate and add a new static chart once chart is ready.
+      google.visualization.events.addListener(chart, 'ready', function () {
+        addStaticChart(mode, chart);
```

Comment:
Thanks - I refactored that out to a method and missed that typo. Fixed and retested.